resources/providers/azurelib/inventory/asset.go

```diff
@@ -38,6 +38,7 @@ const (
 	NetworkSecurityGroup               = "microsoft.network/networksecuritygroups"
 	PostgreSQLDBAssetType              = "microsoft.dbforpostgresql/servers"
 	FlexiblePostgreSQLDBAssetType      = "microsoft.dbforpostgresql/flexibleservers"
+	SecurityContacts                   = "microsoft.security/securitycontacts"
```

Comment:
Maybe better stick to the `AssetType` sufix pattern? 

If you change this code, could you also fix the NetworkSecurityGroup one? 

(my mistake 😬 )

Reply:
sure